src/index.js

```diff
@@ -43,12 +43,24 @@ fetch.mockResponse = (body, init) => {
   );
 };
 
+fetch.mockReject = (body, init) => {
```

Comment:
This is weird, these lines are not in my PR... Keen to hear how to cleanly introduce a good rejection response. I don't know the fetch spec well enough to come off the top for that...